### PR TITLE
Fix url in doc

### PR DESCRIPTION
**Pull request by hkollmann**
To fix the links in doc

### CODE REVIEW

**Comment by oetiker** [comment]
I think that would be one of these instances where squashing would be nice do you agree @derrell ?

**Comment by oetiker** [comment]
@hkollmann can you please also create a PR for the release_5_0_x

**Comment by derrell** [comment]
Indeed. Very appropriate here.

File: documentation/manual/source/_theme.indigo/layout.html [hunk 1]
```diff
@@ -152,83 +147,56 @@
...
-          </li>
-          <li>
-            <a href="http://qooxdoo.org/download">Download</a>
-          </li>
-          <li>
-            <a href="http://news.qooxdoo.org">Blog</a>
+            <a class="" href="http://qooxdoo.org/demos.html">demos</a>
           </li>
           <li>
-            <a href="http://bugs.qooxdoo.org">Bugs</a>
```
Comment (derrell) [inline documentation/manual/source/_theme.indigo/layout.html]: It looks like we lost Bugs, which should probably point to the github Issues page

**Comment by hkollmann** [comment]
@oetiker: How?

**Comment by hkollmann** [comment]
Tried it - hopefully it's correct

**Comment by hkollmann** [comment]
I like the idea to automate the Building of the doc page. Any idea how? And when? I suggest the manual Change for the first try - then automate it

**Comment by hkollmann** [comment]
Is there any way to shorten the merge history?

**Comment by johnspackman** [comment]
This seems to be the same as https://github.com/qooxdoo/qooxdoo/pull/9110, why do we need two pull requests?

**Comment by woprandi** [comment]
@johnspackman To run tests suite on both I guess

**Comment by cajus** [comment]
@johnspackman they target different branches. As they fix the sphinx generated documentation, the commits fix the qx manual. It is regenerated per branch, so you need two PRs. One for master, one for 5_0_*.

**Comment by cajus** [comment]
@hkollmann the docs are automatically built by travis already. Not the PRs, though.

**Comment by oetiker** [comment]
@hkollmann here is some info on rebasing and squashing your commits ... https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

File: documentation/manual/source/_theme.indigo/layout.html [hunk 2]
```diff
@@ -32,25 +32,20 @@
...
     <nav>
       <ul>
-        <!--
         <li>
-          <a class="" href="http://qooxdoo.org/">home</a>
+          <a class="" href="/demos.html">demos</a>
         </li>
-        -->
         <li>
-          <a href="http://news.qooxdoo.org">blog</a>
```
Comment (johnspackman) [inline documentation/manual/source/_theme.indigo/layout.html]: The Blog still exists, at https://medium.com/qooxdoo

**Comment by johnspackman** [comment]
@cajus ok ISWYM but that's different to how I thought we were going to do it, which was to mark each PR that is to be in 5.0.2 with release_5_0_2 milestone, and then use this to update the branch_5_0_x and create the release_5_0_2 tag.

Creating separate PRs for each previous PR to merge into the branch would work but it seems like double the effort when it comes to reviewing and approving something that's already been reviewed and approved.  

I definitely think that in cases like this where the content is the same (or largely the same) the second PR should not be created until the first PR has been approved, otherwise which one do we comment on?  I commented on the other PR before I saw this one, but it's the same commit!

**Comment by oetiker** [comment]
@johnspackman I suggested to create the second PR in this instance primarily to get the website fixed asap ... I think an approach where the initial PR is for master, gets discussed and when agreement is reached, the author creates an additional PR for the active branch ... OR maybe even branches. Since if we follow semantic versioning, we would have bugfixes going into the current branch and features going into the next minor version and breaking changes into master only.

**Comment by cajus** [comment]
@johnspackman, you're right. I didn't compare the commit IDs. Seem to be completely the same. Would be something to do cherry pick.

File: documentation/manual/source/_theme.indigo/layout.html (hunk 2 above)
Comment (hkollmann) [inline documentation/manual/source/_theme.indigo/layout.html]: You are right - but the blog is missing in www.qooxdoo.org either. I copied the footer section from there to have both equals.

Comment (oetiker) [inline documentation/manual/source/_theme.indigo/layout.html]: the blog, btw will soon exist on news.qooxdoo.org again ... medium is setting this up for us

**Comment by hkollmann** [comment]
I can add the Blog entry in 2 weeks - holidays since today. Andy no computer allowed in vacancies. Or could someone else add it?

**Comment by hkollmann** [comment]
@oetiker: Thanks for the hint. Next Time:-)

File: documentation/manual/source/_theme.indigo/layout.html [hunk 3]
```diff
@@ -42,13 +45,17 @@
           <a class="active" href="/current">docs</a>
         </li>
         <li>
+         <a class="" href="/current/apiviewer">api</a>
+        </li>
+        <li>
           <a class="" href="http://stackoverflow.com/questions/tagged/qooxdoo">support</a>
         </li>
         <li>
           <a class="" href="https://gitter.im/qooxdoo/qooxdoo">gitter</a>
```
Comment (derrell) [inline documentation/manual/source/_theme.indigo/layout.html]: It's not one of the PR's changed lines, but...

I suspect that few know what gitter is. How about changing the visible text to "chat room" or "interactive discussion" or some such thing?

Comment (oetiker) [inline documentation/manual/source/_theme.indigo/layout.html]: good idea

**Comment by derrell** [comment]
CALL FOR VOTES

**Pull request merged**